rest/src/test/java/org/envirocar/server/rest/util/OSMTileRendererTests.java

```diff
@@ -29,4 +34,65 @@ public void SaveImageWorks() throws IOException {
 			osmt.saveImage(new BufferedImage(768, 512, BufferedImage.TYPE_INT_RGB), "1235456");
 	}
 	
+	@Test
+	public void zoomLevelTest(){
+		int level=osmt.getZoomLevel(populateCoordinates());
```

Comment:
this basically tests only if you get a zoom level between 0 and 19. An improved test would make use of a priori knowledge: "for these exact coordinates I know what zoom level is the result."
The `populateCoordinates` method should provide static code-defined coordinates. Thus you can ensure that the algorithm stays the way you want it to be, even if you apply changes.